semgrep-core/matching/Generic_vs_generic.ml

```diff
@@ -1196,16 +1206,10 @@ and m_arguments_concat a b =
 
   (* specific case: f"...{$X}..." will properly extract $X from f"foo {bar} baz" *)
   | A.Arg (A.L (A.String("...", a)))::xsa, B.Arg(bexpr)::xsb ->
-      (match Normalize_generic.constant_propagation_and_evaluate_literal bexpr
-       with
-       | Some _ ->
-           (* can match nothing *)
-           (m_arguments_concat xsa xsb) >||>
-           (* can match more *)
-           (m_arguments_concat ((A.Arg (A.L (A.String("...", a))))::xsa) xsb)
-       | None ->
-           (m_arguments_concat xsa (B.Arg(bexpr)::xsb))
-      )
+      (* can match nothing *)
```

Comment:
could you put in comment the call to Normalize_generic.consta... 
and explain why we don't need it anymore now that we have the right constant_propagation?

Reply:
I don't fully understand why it was needed before. However I found the original commit: e775c1df4f036783f0cbbb1097ad0641578c4358.  Maybe it's a good idea that @ulziibay reviews the changes to `Generic_vs_generic.m_arguments_concat` and shed some light on this issue?

Reply:
ulzii is not anymore in the company.
I'll try to merge it today and fix the last issues.